EKS_DISTRO_BASE_TAG_FILE

```diff
@@ -1 +1 @@
-2021-03-22-1616436036
```

Comment:
is this meant for the release tag ? is this generated manually... any plan on build-system doing it ?

Reply:
Yes this was a tagging scheme with date and Unix epoch, that we came up with so that we can identify when each new image was built and the ordering. This is especially since there is a periodic Prowjob that may also update this base image. The other image builds use commit hash Prow environment variables generated from PR's and merges , but the periodic does not have that, so to keep the tagging consistent within this image build, we chose this.

Reply:
So the plan is to have an automation that will update these, when a new image is built. Automated jobs would cut a PR and some human would approve it and merge it. Since these are private repos now, we are punting the automation tasks for now until we will be public.

The number of automation stuff is starting to grow, I'm going to start an umbrella issue that would track all the automation we have to do once we go public.

Reply:
+1
I wanted to do some testing on generating automated PR's for private repos too. It's just a matter of cloning them properly and seeing if Github CLI will support the operations on private repos.

Reply:
Thanks guys for clarifying.